evennia/contrib/ingame_python/scripts.py

```diff
@@ -115,7 +115,11 @@ def get_events(self, obj):
         events = {}
         all_events = self.ndb.events
         classes = Queue()
-        classes.put(type(obj))
+        if isinstance(obj, ObjectDB):
```

Comment:
`isinstance` will only detect if you pass the actual `ObjectDB` class in there. Use `utils.utils.inherits_from` instead, that will catch any class inheriting at any distance from `ObjectDB`.